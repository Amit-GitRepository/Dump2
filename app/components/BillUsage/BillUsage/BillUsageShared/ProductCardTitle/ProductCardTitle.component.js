import Proptypes from 'prop-types';
import React, {Component} from 'react';
import styles from './ProductCardTitle.style';
import {Icon, ISText, Touchable} from '../../../../Shared';
import {View} from 'react-native';

class ProductCardTitle extends Component {

  render () {
    const {text, subtext, isPrepaid, navigate} = this.props;
    const subTextStyles = (isPrepaid) ? styles.prepaidSubText : styles.subtext;
    return (
      <View style={styles.container}>
        <View style={styles.textArea}>
          <View style={styles.container}>
            <ISText type='BOLD' style={styles.text} numberOfLines={1} ellipsizeMode='tail'>{text}</ISText>
            { navigate && <Touchable borderless={true} onPress={navigate}><Icon style={styles.icon} name='bills_usage_search' size={25}/></Touchable>}
          </View>
          {subtext ? <ISText type='SEMIBOLD' style={subTextStyles}>{subtext}</ISText> : null}
        </View>

      </View>
    );
  }
}
ProductCardTitle.defaultProps = {
  text: '',
  subtext: '',
  isPrepaid: false
};
ProductCardTitle.propTypes = {
  text: Proptypes.string,
  subtext: Proptypes.string,
  navigate: Proptypes.func,
  isPrepaid: Proptypes.bool
};
export default ProductCardTitle;
