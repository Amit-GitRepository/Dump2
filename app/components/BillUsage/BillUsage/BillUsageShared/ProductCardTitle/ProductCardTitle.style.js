import {colors, fonts} from '../../../../../themes/constants.styles';

export default {
  container: {
    flexDirection: 'row',
    paddingRight: 28,
    paddingLeft: 25
  },
  textArea: {
    alignItems: 'center'
  },
  text: {
    fontSize: fonts.FONT_SIZE_NORMAL
  },
  subtext: {
    color: colors.PRIMARY_ACTIONABLE,
    fontSize: fonts.FONT_SIZE_SMALL
  },
  prepaidSubText: {
    color: colors.PRIMARY_TEXT_TAB_LABEL
  },
  icon: {
    marginLeft: 5,
    alignSelf: 'flex-start',
    color: colors.PRIMARY_BG_TEXT_CONTRAST,
    padding: 3,
    fontSize: fonts.FONT_SIZE_NORMAL,
    backgroundColor: colors.PRIMARY_ACTIONABLE,
    borderWidth: 1,
    borderColor: colors.PRIMARY_ACTIONABLE,
    borderRadius: 12,
    overflow: 'hidden'
  }
};
