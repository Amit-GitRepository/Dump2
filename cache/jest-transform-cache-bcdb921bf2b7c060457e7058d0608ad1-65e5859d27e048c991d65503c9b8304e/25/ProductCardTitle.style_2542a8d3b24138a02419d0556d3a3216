Object.defineProperty(exports,"__esModule",{value:true});var cov_1poaco7qk6=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/ProductCardTitle/ProductCardTitle.style.js',hash='0a205708df284cc1264cca576a0af90a9a1acf36',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/ProductCardTitle/ProductCardTitle.style.js',statementMap:{},fnMap:{},branchMap:{},s:{},f:{},b:{},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _constants=require('../../../../../themes/constants.styles');exports.default=

{
container:{
flexDirection:'row'},

textArea:{
alignItems:'center'},

text:{
fontSize:_constants.fonts.FONT_SIZE_NORMAL},

subtext:{
color:_constants.colors.PRIMARY_ACTIONABLE,
fontSize:_constants.fonts.FONT_SIZE_SMALL},

prepaidSubText:{
color:_constants.colors.PRIMARY_TEXT_TAB_LABEL},

icon:{
marginLeft:5,
alignSelf:'flex-start',
color:_constants.colors.PRIMARY_BG_TEXT_CONTRAST,
padding:3,
fontSize:_constants.fonts.FONT_SIZE_NORMAL,
backgroundColor:_constants.colors.PRIMARY_ACTIONABLE,
borderWidth:1,
borderColor:_constants.colors.PRIMARY_ACTIONABLE,
borderRadius:12,
overflow:'hidden'}};