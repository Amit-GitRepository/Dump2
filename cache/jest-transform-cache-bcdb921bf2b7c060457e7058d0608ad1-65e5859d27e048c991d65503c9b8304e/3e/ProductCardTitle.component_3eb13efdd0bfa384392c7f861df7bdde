Object.defineProperty(exports,"__esModule",{value:true});var cov_215ydcnhmb=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/ProductCardTitle/ProductCardTitle.component.js',hash='2485d5469645425f77f9a28e4e31d0531f8adee0',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/ProductCardTitle/ProductCardTitle.component.js',statementMap:{'0':{start:{line:13,column:39},end:{line:13,column:49}},'1':{start:{line:14,column:26},end:{line:14,column:78}},'2':{start:{line:15,column:4},end:{line:26,column:6}},'3':{start:{line:29,column:0},end:{line:34,column:2}},'4':{start:{line:35,column:0},end:{line:40,column:2}}},fnMap:{'0':{name:'(anonymous_0)',decl:{start:{line:12,column:2},end:{line:12,column:3}},loc:{start:{line:12,column:12},end:{line:27,column:3}},line:12}},branchMap:{'0':{loc:{start:{line:14,column:26},end:{line:14,column:78}},type:'cond-expr',locations:[{start:{line:14,column:40},end:{line:14,column:61}},{start:{line:14,column:64},end:{line:14,column:78}}],line:14}},s:{'0':0,'1':0,'2':0,'3':0,'4':0},f:{'0':0},b:{'0':[0,0]},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _createClass=function(){function defineProperties(target,props){for(var i=0;i<props.length;i++){var descriptor=props[i];descriptor.enumerable=descriptor.enumerable||false;descriptor.configurable=true;if("value"in descriptor)descriptor.writable=true;Object.defineProperty(target,descriptor.key,descriptor);}}return function(Constructor,protoProps,staticProps){if(protoProps)defineProperties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor;};}();var _Icon=require('../../../../Shared/Icon/Icon.component');var _Icon2=_interopRequireDefault(_Icon);
var _ISText=require('../../../../Shared/ISText/ISText.component');var _ISText2=_interopRequireDefault(_ISText);
var _noop=require('lodash/noop');var _noop2=_interopRequireDefault(_noop);
var _propTypes=require('prop-types');var _propTypes2=_interopRequireDefault(_propTypes);
var _react=require('react');var _react2=_interopRequireDefault(_react);
var _ProductCardTitle=require('./ProductCardTitle.style');var _ProductCardTitle2=_interopRequireDefault(_ProductCardTitle);
var _Touchable=require('../../../../Shared/Touchable/Touchable.component');var _Touchable2=_interopRequireDefault(_Touchable);
var _reactNative=require('react-native');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call&&(typeof call==="object"||typeof call==="function")?call:self;}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+typeof superClass);}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass;}var

ProductCardTitle=function(_Component){_inherits(ProductCardTitle,_Component);function ProductCardTitle(){_classCallCheck(this,ProductCardTitle);return _possibleConstructorReturn(this,(ProductCardTitle.__proto__||Object.getPrototypeOf(ProductCardTitle)).apply(this,arguments));}_createClass(ProductCardTitle,[{key:'render',value:function render()

{cov_215ydcnhmb.f[0]++;var _ref=(cov_215ydcnhmb.s[0]++,
this.props),text=_ref.text,subtext=_ref.subtext,isPrepaid=_ref.isPrepaid;
var subTextStyles=(cov_215ydcnhmb.s[1]++,isPrepaid?(cov_215ydcnhmb.b[0][0]++,_ProductCardTitle2.default.prepaidSubText):(cov_215ydcnhmb.b[0][1]++,_ProductCardTitle2.default.subtext));cov_215ydcnhmb.s[2]++;
return(
_react2.default.createElement(_reactNative.View,{style:_ProductCardTitle2.default.container},
_react2.default.createElement(_reactNative.View,{style:_ProductCardTitle2.default.textArea},
_react2.default.createElement(_reactNative.View,{style:_ProductCardTitle2.default.container},
_react2.default.createElement(_ISText2.default,{type:'BOLD',style:_ProductCardTitle2.default.text},text),
_react2.default.createElement(_Touchable2.default,{onPress:this.props.navigate},_react2.default.createElement(_Icon2.default,{style:_ProductCardTitle2.default.icon,name:'bills_usage_search',size:25}))),

_react2.default.createElement(_ISText2.default,{type:'SEMIBOLD',style:subTextStyles},subtext))));




}}]);return ProductCardTitle;}(_react.Component);cov_215ydcnhmb.s[3]++;

ProductCardTitle.defaultProps={
text:'',
subtext:'',
navigate:_noop2.default,
isPrepaid:false};cov_215ydcnhmb.s[4]++;

ProductCardTitle.propTypes={
text:_propTypes2.default.string,
subtext:_propTypes2.default.string,
navigate:_propTypes2.default.func,
isPrepaid:_propTypes2.default.bool};exports.default=

ProductCardTitle;