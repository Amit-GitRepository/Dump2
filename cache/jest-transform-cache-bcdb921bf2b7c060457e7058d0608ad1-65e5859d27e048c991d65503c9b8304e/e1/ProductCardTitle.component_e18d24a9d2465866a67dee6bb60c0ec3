Object.defineProperty(exports,"__esModule",{value:true});var cov_215ydcnhmb=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/ProductCardTitle/ProductCardTitle.component.js',hash='277cdbdb18ba44bb7e8876b76b49bb3c5607ff68',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/ProductCardTitle/ProductCardTitle.component.js',statementMap:{'0':{start:{line:10,column:49},end:{line:10,column:59}},'1':{start:{line:11,column:26},end:{line:11,column:78}},'2':{start:{line:12,column:4},end:{line:23,column:6}},'3':{start:{line:26,column:0},end:{line:30,column:2}},'4':{start:{line:31,column:0},end:{line:36,column:2}}},fnMap:{'0':{name:'(anonymous_0)',decl:{start:{line:9,column:2},end:{line:9,column:3}},loc:{start:{line:9,column:12},end:{line:24,column:3}},line:9}},branchMap:{'0':{loc:{start:{line:11,column:26},end:{line:11,column:78}},type:'cond-expr',locations:[{start:{line:11,column:40},end:{line:11,column:61}},{start:{line:11,column:64},end:{line:11,column:78}}],line:11},'1':{loc:{start:{line:17,column:14},end:{line:17,column:131}},type:'binary-expr',locations:[{start:{line:17,column:14},end:{line:17,column:22}},{start:{line:17,column:26},end:{line:17,column:131}}],line:17}},s:{'0':0,'1':0,'2':0,'3':0,'4':0},f:{'0':0},b:{'0':[0,0],'1':[0,0]},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _createClass=function(){function defineProperties(target,props){for(var i=0;i<props.length;i++){var descriptor=props[i];descriptor.enumerable=descriptor.enumerable||false;descriptor.configurable=true;if("value"in descriptor)descriptor.writable=true;Object.defineProperty(target,descriptor.key,descriptor);}}return function(Constructor,protoProps,staticProps){if(protoProps)defineProperties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor;};}();var _propTypes=require('prop-types');var _propTypes2=_interopRequireDefault(_propTypes);
var _react=require('react');var _react2=_interopRequireDefault(_react);
var _ProductCardTitle=require('./ProductCardTitle.style');var _ProductCardTitle2=_interopRequireDefault(_ProductCardTitle);
var _Shared=require('../../../../Shared');
var _reactNative=require('react-native');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call&&(typeof call==="object"||typeof call==="function")?call:self;}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+typeof superClass);}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass;}var

ProductCardTitle=function(_Component){_inherits(ProductCardTitle,_Component);function ProductCardTitle(){_classCallCheck(this,ProductCardTitle);return _possibleConstructorReturn(this,(ProductCardTitle.__proto__||Object.getPrototypeOf(ProductCardTitle)).apply(this,arguments));}_createClass(ProductCardTitle,[{key:'render',value:function render()

{cov_215ydcnhmb.f[0]++;var _ref=(cov_215ydcnhmb.s[0]++,
this.props),text=_ref.text,subtext=_ref.subtext,isPrepaid=_ref.isPrepaid,navigate=_ref.navigate;
var subTextStyles=(cov_215ydcnhmb.s[1]++,isPrepaid?(cov_215ydcnhmb.b[0][0]++,_ProductCardTitle2.default.prepaidSubText):(cov_215ydcnhmb.b[0][1]++,_ProductCardTitle2.default.subtext));cov_215ydcnhmb.s[2]++;
return(
_react2.default.createElement(_reactNative.View,{style:_ProductCardTitle2.default.container},
_react2.default.createElement(_reactNative.View,{style:_ProductCardTitle2.default.textArea},
_react2.default.createElement(_reactNative.View,{style:_ProductCardTitle2.default.container},
_react2.default.createElement(_Shared.ISText,{type:'BOLD',style:_ProductCardTitle2.default.text},text),
(cov_215ydcnhmb.b[1][0]++,navigate)&&(cov_215ydcnhmb.b[1][1]++,_react2.default.createElement(_Shared.Touchable,{onPress:navigate},_react2.default.createElement(_Shared.Icon,{style:_ProductCardTitle2.default.icon,name:'bills_usage_search',size:25})))),

_react2.default.createElement(_Shared.ISText,{type:'SEMIBOLD',style:subTextStyles},subtext))));




}}]);return ProductCardTitle;}(_react.Component);cov_215ydcnhmb.s[3]++;

ProductCardTitle.defaultProps={
text:'',
subtext:'',
isPrepaid:false};cov_215ydcnhmb.s[4]++;

ProductCardTitle.propTypes={
text:_propTypes2.default.string,
subtext:_propTypes2.default.string,
navigate:_propTypes2.default.func,
isPrepaid:_propTypes2.default.bool};exports.default=

ProductCardTitle;