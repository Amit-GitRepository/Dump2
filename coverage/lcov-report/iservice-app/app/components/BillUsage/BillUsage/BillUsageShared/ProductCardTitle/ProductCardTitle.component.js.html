<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/ProductCardTitle/ProductCardTitle.component.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../../../../../index.html">All files</a> / <a href="index.html">iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/ProductCardTitle</a> ProductCardTitle.component.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>5/5</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>4/4</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>5/5</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">10x</span>
<span class="cline-any cline-yes">10x</span>
<span class="cline-any cline-yes">10x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">50x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">50x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import Proptypes from 'prop-types';
import React, {Component} from 'react';
import styles from './ProductCardTitle.style';
import {Icon, ISText, Touchable} from '../../../../Shared';
import {View} from 'react-native';
&nbsp;
class ProductCardTitle extends Component {
&nbsp;
  render () {
    const {text, subtext, isPrepaid, navigate} = this.props;
    const subTextStyles = (isPrepaid) ? styles.prepaidSubText : styles.subtext;
    return (
      &lt;View style={styles.container}&gt;
        &lt;View style={styles.textArea}&gt;
          &lt;View style={styles.container}&gt;
            &lt;ISText type='BOLD' style={styles.text}&gt;{text}&lt;/ISText&gt;
            { navigate &amp;&amp; &lt;Touchable onPress={navigate}&gt;&lt;Icon style={styles.icon} name='bills_usage_search' size={25}/&gt;&lt;/Touchable&gt;}
          &lt;/View&gt;
          &lt;ISText type='SEMIBOLD' style={subTextStyles}&gt;{subtext}&lt;/ISText&gt;
        &lt;/View&gt;
&nbsp;
      &lt;/View&gt;
    );
  }
}
ProductCardTitle.defaultProps = {
  text: '',
  subtext: '',
  isPrepaid: false
};
ProductCardTitle.propTypes = {
  text: Proptypes.string,
  subtext: Proptypes.string,
  navigate: Proptypes.func,
  isPrepaid: Proptypes.bool
};
export default ProductCardTitle;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Sun Dec 03 2017 16:14:06 GMT+0530 (IST)
</div>
</div>
<script src="../../../../../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../../../../../sorter.js"></script>
</body>
</html>
