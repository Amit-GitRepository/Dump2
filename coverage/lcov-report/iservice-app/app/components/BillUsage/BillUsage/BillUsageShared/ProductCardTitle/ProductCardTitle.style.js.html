<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/ProductCardTitle/ProductCardTitle.style.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../../../../../index.html">All files</a> / <a href="index.html">iservice-app/app/components/BillUsage/BillUsage/BillUsageShared/ProductCardTitle</a> ProductCardTitle.style.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>0/0</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import {colors, fonts} from '../../../../../themes/constants.styles';
&nbsp;
export default {
  container: {
    flexDirection: 'row'
  },
  textArea: {
    alignItems: 'center'
  },
  text: {
    fontSize: fonts.FONT_SIZE_NORMAL
  },
  subtext: {
    color: colors.PRIMARY_ACTIONABLE,
    fontSize: fonts.FONT_SIZE_SMALL
  },
  prepaidSubText: {
    color: colors.PRIMARY_TEXT_TAB_LABEL
  },
  icon: {
    marginLeft: 5,
    alignSelf: 'flex-start',
    color: colors.PRIMARY_BG_TEXT_CONTRAST,
    padding: 3,
    fontSize: fonts.FONT_SIZE_NORMAL,
    backgroundColor: colors.PRIMARY_ACTIONABLE,
    borderWidth: 1,
    borderColor: colors.PRIMARY_ACTIONABLE,
    borderRadius: 12,
    overflow: 'hidden'
  }
};
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Sun Dec 03 2017 16:14:06 GMT+0530 (IST)
</div>
</div>
<script src="../../../../../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../../../../../sorter.js"></script>
</body>
</html>
