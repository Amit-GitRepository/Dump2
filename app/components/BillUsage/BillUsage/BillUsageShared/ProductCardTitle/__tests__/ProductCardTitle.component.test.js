import ProductCardTitle from '../ProductCardTitle.component';
import React from 'react';
import renderer from 'react-test-renderer'; // Enzyme adapter for React 16

describe('ProductCardTitle component', () => {
  it('renders correctly', () => {
    const component = renderer.create(<ProductCardTitle />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
