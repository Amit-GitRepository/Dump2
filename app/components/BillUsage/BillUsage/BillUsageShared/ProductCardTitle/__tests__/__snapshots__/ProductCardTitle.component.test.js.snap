// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductCardTitle component renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "paddingLeft": 25,
      "paddingRight": 28,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingLeft": 25,
          "paddingRight": 28,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={1}
        style={
          Object {
            "backgroundColor": "transparent",
            "color": "#303C46",
            "fontFamily": "SukhumvitSet-Bold",
            "fontSize": 16,
          }
        }
      >
        
      </Text>
    </View>
  </View>
</View>
`;
